app/models/skin.rb

```diff
@@ -111,7 +111,7 @@ def valid_public_preview
   end
 
   validates_presence_of :title
-  validates_uniqueness_of :title, message: ts('must be unique')
+  validates_uniqueness_of :title, message: ts('must be unique'), case_sensitive: false
```

Comment:
ticking pointed out this will fix [AO3-5918](https://otwarchive.atlassian.net/browse/AO3-5918), but it'll leave us with skins with invalid titles. Do we want to change this to `case_sensitive: true` until we can deal with the existing skins, or do we have faith in our users' ability to change the names of their skins if they encounter a validation error?

Reply:
Let's set it to true and keep 5918 out for now.